src/app.js

```diff
@@ -0,0 +1,139 @@
+import {div} from '@cycle/dom';
+import _ from 'lodash';
+import xs from 'xstream';
+
+// TO DO:
+// X Render Hello World
+// X Render board
+// X Render knight
+// X Move knight to legal squares by clicking
+// - Move knight to legal squares by dragging
+//
+
+function Board () {
+  return  _.range(0, 8).map(() =>
+    _.range(0, 8).map((square) => Square())
+  );
+}
+
+function Square () {
+  return {}
+}
+
+function isBlack (rowIndex, squareIndex) {
+  if (rowIndex % 2 === 0 && squareIndex % 2 !== 0) {
+    return true;
+  } else if (rowIndex % 2 !== 0 && squareIndex % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function hasKnight (knightPosition, rowIndex, squareIndex) {
+  if (rowIndex === knightPosition.x && squareIndex === knightPosition.y) {
```

Comment:
I think this form is a little more readable:

`return rowIndex === knightPosition.x && squareIndex === knightPosition.y;`